tests/unit/ingredients_extract.t

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl -w
+
+# Tests of Ingredients::preparse_ingredients_text()
+
+use Modern::Perl '2017';
+use utf8;
+
+use Test::More;
+use Log::Any::Adapter 'TAP';
+
+use ProductOpener::Products qw/:all/;
+use ProductOpener::Tags qw/:all/;
+use ProductOpener::TagsEntries qw/:all/;
+use ProductOpener::Ingredients qw/:all/;
+
+my @lists = (
+	# en phrases_after_ingredients_list
+	[
+		"en",
+		"carrots, green peas, corn, scallion. *adds a trivial amount of added sugars per serving.",
+		"carrots, green peas, corn, scallion.",
+	],
+	# en ignore_phrases,
+	[
+		"en",
+		"Egg White, Contains 2% or less of: Xanthan Gum, Salt, Glucono-delta-lactone.",
```

Comment:
I think we should keep this test, but that we should keep the "Contains 2%..." part.